test/azlinux_test.go

```diff
@@ -528,7 +530,16 @@ echo "$BAR" > bar.txt
 			)
 			sr.Evaluate = true
 
-			solveT(ctx, t, gwc, sr)
+			beforeBuild := time.Now()
+			res := solveT(ctx, t, gwc, sr)
+
+			dt, ok := res.Metadata[exptypes.ExporterImageConfigKey]
+			assert.Assert(t, ok, "result metadata should contain an image config: available metadata: %s", strings.Join(maps.Keys(res.Metadata), ", "))
+
+			var cfg dalec.DockerImageSpec
+			assert.Assert(t, json.Unmarshal(dt, &cfg))
+			assert.Check(t, cfg.Created.After(beforeBuild))
+			assert.Check(t, cfg.Created.Before(time.Now()))
```

Comment:
should we do a similar check in debian tests?

Reply:
Debian/Ubuntu is running these tests.